Headers/Internal/VLCRendererItem+Init.h

```diff
@@ -41,6 +41,6 @@
  * Returns the C renderer item
  * \return Renderer item
  */
-- (libvlc_renderer_item_t * _Nonnull)rendererItem;
```

Comment:
I understand why you did that but since this is part of your wrapper and VLCKit the CamelCase is and was absolutely correct here

Reply:
Good Points. The naming conflict should be fine since it's a property on the rendereritem but I see how that could be confusing. For the c pointer argument, I'm trying to see if we have that case somewhere else in VLCkit and how we handled that or try to find Apple code

Reply:
I think the libVLCMediaDescriptor on VLCMedia might be a comparable example. We have in the past used void * for any c pointers. You can find the same behavior in VLCMediaPlayer with the internal variable _playerInstance.

Reply:
I don't mind renaming it but isn't `rendererItem.libvlcRendererItem` a little verbosy? 😄

Reply:
Lololol :D may I remind you that you're coding in Objective-C ? :D That's nothing!
*Looks over to CMAttitudeReferenceFrameXArbitraryCorrectedZVertical*

Reply:
lol that's a very good point!

Reply:
Sorry for being late to the party, but using a void pointer would be the correct way to do it.

Reply:
Even if we're not exposing it?

Reply:
That would follow the logic of the internal VLCLibVLCbridging.h header, yes.